composer.json

```diff
@@ -20,16 +20,16 @@
     "prefer-stable": true,
     "require": {
         "php": "^7.4|^8.0",
-        "foxy/foxy": "^1.1@dev",
+        "wikimedia/composer-merge-plugin": "dev-feature/composer-v2",
```

Comment:
Is that needed? Yii 2 worked w/o it relying on the same asset packagist well.

Reply:
done.